src/fmdb/FMDatabaseQueue.m

```diff
@@ -137,6 +137,10 @@ - (FMDatabase*)database {
 }
 
 - (void)inDatabase:(void (^)(FMDatabase *db))block {
+    [self inDatabase:block async:NO];
+}
+
+- (void)inDatabase:(void (^)(FMDatabase *db))block async:(BOOL)async {
     /* Get the currently executing queue (which should probably be nil, but in theory could be another DB queue
      * and then check it against self to make sure we're not about to deadlock. */
```

Comment:
I agree, let's keep the differences to the upstream master as minimal as possible.